console-api/proto/tasks.proto

```diff
@@ -130,6 +130,13 @@ message Stats {
     common.PollStats poll_stats = 7;
     // The total number of times this task has woken itself.
     uint64 self_wakes = 8;
+    // The total duration this task was scheduled prior to being polled, summed
+    // across all poll cycles. Note that this includes only polls that have
+    // started and is not reflecting any scheduled state where the task hasn't
+    // yet been polled. Subtracting both `busy_time` and `scheduled_time` from the
+    // total lifetime of the polled object results in the amount of time it spent
```

Comment:
nit: since this is now only defined on tasks, we can just say "task" here:
```suggestion
    // total lifetime of the polled task results in the amount of time it spent
```